backend/src/repo/game_repo.rs

```diff
@@ -6,7 +6,7 @@ pub trait GameRepo {
 
 #[derive(Clone)]
 pub struct InMemoryGameRepo {
-    games: Vec<Game>,
+    pub(crate) games: Vec<Game>,
```

Comment:
určitě ne public. Pokud bychom ukládali hry do databáze, tak `PostgresGameRepo` přece žádné pole `games` mít nebude. Potřebuješ vytvořit metodu, která ti bude zajišťovat získávání her.